Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml

```
@@ -0,0 +1,172 @@
+commonfields:
+  id: ArcusTeam
+  version: -1
+name: ArcusTeam
+display: ArcusTeam
+category: Data Enrichment & Threat Intelligence
+description: 'The ArcusTeam API allows the user to inspect connected devices'' attack
+  surface. By feeding device identifiers and the software it runs: DeviceTotal will
+  return a map of the device’s attack surface. DeviceTotal was built from the ground
+  up in order to provide complete visibility into connected devices and mitigate 3rd
+  party risk. DeviceTotal can continuously identify & predict such that the connected
+  device security posture is being assessed, prioritized and mitigated effectively.'
+configuration:
+- display: Server URL (e.g. https://web.prod.arcusteam.com)
+  name: url
+  defaultvalue: https://web.prod.arcusteam.com
+  type: 0
+  required: true
+  additionalinfo: The FQDN/IP the integration should connect to.
+- display: API Key
+  name: api_key
+  type: 4
+  required: true
+  additionalinfo: ' The API Key required to authenticate to the service.'
+- display: 'The client ID'
+  name: client_id
+  type: 0
+  required: true
+  additionalinfo: The client ID from ArcusTeam dashboard
```

Adding insecure and proxy params
```suggestion
  additionalinfo: The client ID from ArcusTeam dashboard
- display: Trust any certificate (not secure)
  name: insecure
  type: 8
  required: false
- display: Use system proxy settings
  name: proxy
  type: 8
  required: false
```